lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -9,13 +11,23 @@
 file_model_disp = os.path.join(output_dir, 'reg_disp_vector.sav')
 file_model_gh_sep = os.path.join(output_dir, 'cls_gh.sav')
 
+
+
 def test_lstchain_mc_r0_to_dl1():
     input_file = mc_gamma_testfile
     cmd = f'lstchain_mc_r0_to_dl1 -f {input_file} -o {output_dir}'
     os.system(cmd)
     assert os.path.exists(dl1_file)
 
 
+@pytest.mark.run(after='test_lstchain_mc_r0_to_dl1')
+def test_add_source_dependent_parameters():
+    cmd = f'lstchain_add_source_dependent_parameters -f {dl1_file}'
+    os.system(cmd)
```

Comment:
I don't think pytest realizes that this counts towards coverage. So better keep the other test ;)

Reply:
ok, done:)
Now all checks have passed!